samples/buildSrc-test-junit4/buildSrc/src/main/kotlin/tools/BuildHelper.kt

```diff
@@ -0,0 +1,3 @@
+package tools
+
+fun sayHelloTo(name: String) = "Hello $name"
```

Comment:
The class of that name (and package) is a little bit missleading when here is just an "sayHello" function inside.
Renaming the package to `greeting` and the filename to `Greeter.kt`.
As I remember correctly Gradle use these convention in mutliple other samples too..

Reply:
done